scripts/rc.d/tmuxinator.bash

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+#Setting for the editor to use with tmuxinator edit <config>
+export EDITOR=$(which vim)
```

Comment:
Why is this added here? What if the user already has `$EDITOR` exported to another term (`nvim`, `emacs`)?

Reply:
It is needed only for editing the configs using the tmuxinator commands, which removes the need for a user to determine the location of the config files themselves. It can have a check to see if editor is already set